packages/components/src/templates/forms.stories.ts

```diff
@@ -0,0 +1,70 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+/**
+ * ```
+ * ```
+ */
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Forms',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3060-14503&t=LHCyCYMjDV22qemZ-4'
+    }
+  }
+};
+
+export const loginForm = {
+  render: () => html`
+    <section class="sd-container">
+      <form>
+        <h3 class="sd-headline sd-headline--size-3xl mb-8">Login for UnionFondsOnline users</h3>
+        <sd-input type="text" label="Access number" spellcheck class="mb-4"></sd-input>
```

Comment:
Include autocompletion (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)  here. If this can only can be a number, please adapt inputmode.

Reply:
Updated